src/config.js

```diff
@@ -1,24 +1,23 @@
 module.exports = {
   emojis: {
-    tick: '<:tick:957713309031858337> ',
-    cross: '<:cross:957713099291504691> ',
-    gold: '<:pokecoin:957706942007115827>',
+    cross: '<:cross:957713099291504691>',
+    favourite: '❤️',
+    gems: '<:gems:957853293923676160>',
+    gold: '<:gold:957706942007115827>',
     redeems: '<:redeems:957706969819521127>',
-    gems: '<:diamond:957853293923676160>',
     shiny: '✨',
-    fav: '❤️',
+    tick: '<:tick:957713309031858337>',
   },
   colors: {
-    default: 0x7289da,
-    red: 0xff0000,
-    green: 0x00ff00,
+    default: 'Blurple',
+    error: 'Red',
+    success: 'Green',
   },
   errors: {
-    confirmationTimeout: `You took too long to confirm`,
-    userNotStarted: `You haven't started yet, please use \`/start\` to choose a starter.`,
-    componentNotRequestedByUser: `You can't use these components as this wasn't requested by you.`,
+    component_not_requested_by_user:
+      "You can't use these components as this wasn't requested by you.",
+    confirmation_timeout: 'You took too long to confirm',
+    user_not_started:
+      "You haven't started yet, please use `/start` to choose a starter.",
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/Shadeslayer10/PokeRealm/pullRequest?prid=9275083)
```suggestion
      'You haven\'t started yet, please use `/start` to choose a starter.',
```